vantiq-cloud-infra-operations/terraform_azure/new/modules/aks/variables.tf

```diff
@@ -41,15 +41,11 @@ variable "dns_service_ip" {
   type        = string
   default     = null
 }
-variable "docker_bridge_cidr" {
-  description = "docker bridge cidr"
-  type        = string
-  default     = "172.17.0.1/16"
-}
+
 variable "load_balancer_sku" {
   description = "load balacer sku. must be standard or higher"
   type        = string
-  default     = "Standard"
+  default     = "standard"
```

Comment:
Default value should be `standard`.
- https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/kubernetes_cluster#load_balancer_sku